x-pack/plugins/cloud_security_posture/public/common/constants.ts

```diff
@@ -263,9 +263,7 @@ The runtime mappings are used to prevent filtering out the data when any of thes
 TODO: Remove the fields below once they are mapped as Keyword in the Third Party integrations, or remove
 the fields from the runtime mappings if they are removed from the Data Table.
 */
-export const CDR_VULNERABILITY_DATA_TABLE_RUNTIME_MAPPING_FIELDS: string[] = [
-  VULNERABILITY_FIELDS.VENDOR,
-];
+export const CDR_VULNERABILITY_DATA_TABLE_RUNTIME_MAPPING_FIELDS: string[] = [];
```

Comment:
decided to keep the constant while we have the runtime mappings for misconfigurations, so its clear that there's no runtime mappings for vulnerabilities.